lineman/spec-e2e/helpers/invitations_helper.coffee

```diff
@@ -18,3 +21,19 @@ module.exports = new class InvitationsHelper
 
   submitInvitationsForm: ->
     element(By.css('.invitation-form__submit')).click()
+
+  visitMembershipsPage: ->
```

Comment:
I would call this 'clickManageMembers' visit tells me you're going to enter a url